library/_text_parser.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
```

Comment:
This file should be renamed `library/_command_parser.py`

Reply:
Oh, wait, is this the backwards-compatibility part? If so, ignore my comment above.